src/main/java/dan200/computercraft/shared/Registry.java

```diff
@@ -376,6 +377,9 @@ private static void registerProviders()
         ComputerCraftAPI.registerGenericCapability( CapabilityEnergy.ENERGY );
         ComputerCraftAPI.registerGenericCapability( CapabilityFluidHandler.FLUID_HANDLER_CAPABILITY );
 
+        // Register detail providers
+        ComputerCraftAPI.registerDetailProvider( new PrintoutItemDetailProvider() );
```

Comment:
Sorry to be a pain, but can we remove this? I think I just want to add the API and nothing else, as it saves me having to think about what's exposed to users (as that has more stability guarantees than the Java API).